src/style/content/conversation/title-bar.less

```diff
@@ -46,6 +46,10 @@
     margin: 4px 16px 0 0;
   }
 }
+
+.conversation-title-bar-legal-hold {
+  margin-right: 6px;
+}
```

Comment:
nothing constructive to add but I just noticed that `cursor: pointer` would be much better at `#show-participants` than `.conversation-title-bar-name`

Reply:
Why do you think so?